packages/thriver-core/lib/templates/canvas.html

```diff
@@ -33,6 +33,8 @@
       {{> feedbackAdmin}}
     {{/if}}
   {{/if}}
+  {{> alerts}}
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.min.js"></script>
```

Comment:
Accidental duplication of the script being loaded on line 39?

Reply:
Also, why do we have this at all?  Doesn't Meteor take care of shivs for old browsers?